src/layers/legacy/medPacs/medAbstractPacsFindScu.cpp

```diff
@@ -15,7 +15,8 @@
 
 #include <dtkCoreSupport/dtkGlobal.h>
 
-int medAbstractPacsFindScu::sendFindRequest(const char* peerTitle, const char* peerIP, unsigned int peerPort, const char* ourTitle, const char* ourIP, unsigned int ourPort)
```

Comment:
One line is better

Reply:
Two lines is the default in these pacs files, this change has been made to unify the coding style.

And, as you can see, long lines are awful in our IDE and in Github Pull Request. It's more complicated to review these long lines, and lead to review errors, and eyes tiredness for the reviewer. It's also more complicated to understand a code when some of the lines are so long that we need to scroll constantly. It is way complicated also if you work in your IDE with several windows opened, as i do, in order to have a global vision of the code you're working on.

If at one point we have a survey about coding style and specially long code lines, i would vote to forbid them :D